Reconstruction/RecCalorimeter/src/components/SingleCaloTowerTool.cpp

```diff
@@ -0,0 +1,183 @@
+#include "SingleCaloTowerTool.h"
```

Comment:
this file probably was removed by some previous PR ? (Hence it appears as if you add it?)